src/Home/HomePage.js

```diff
@@ -0,0 +1,85 @@
+
```

Comment:
Since all the other filenames are hyphen separated, you can follow the same for this as - Home-page.js

Reply:
Yeah i will refactor the naming for all files. and make it consistent